test/component/Utf8Spec.scala

```diff
@@ -60,9 +60,10 @@ class Utf8Spec extends ComponentTestSpec with AuditService with ExpectedTestResp
     stopMockServer()
   }
 
-  val ValidRawXmlByte = ByteString.fromArray(Files.readAllBytes(Paths.get("target/scala-2.13/test-classes/raw/valid_xml.raw")))
-  val InvalidRawXmlByte = ByteString.fromArray(Files.readAllBytes(Paths.get("target/scala-2.13/test-classes/raw/invalid_xml.raw")))
-
+  // TODO: get Scala version number, avoid string literal
```

Comment:
Todo?

Reply:
Completed ToDo, added plugin and config to allow us to fetch build info dynamically in the future.